nekoyume/Assets/_Scripts/UI/Module/Item/EquipmentSlots.cs

```diff
@@ -148,8 +149,9 @@ public bool TryGetToEquip(Costume costume, out EquipmentSlot slot)
         /// </summary>
         /// <param name="equipment"></param>
         /// <param name="slot"></param>
+        /// <param name="isMimisBrunnr">기본적으로는 false이고, 미미르의 샘 준비 UI에서만 장착 규칙이 달라 true를 입력하면 된다.</param>
         /// <returns></returns>
-        public bool TryGetToEquip(Equipment equipment, out EquipmentSlot slot)
+        public bool TryGetToEquip(Equipment equipment, out EquipmentSlot slot, bool isMimisBrunnr = false)
```

Comment:
Rather than receiving the `isMimisBrunnr` argument, it seems better to receive an 'ElementalType' that is intuitive with logic.

Reply:
I checked it and will fix it. thanks.